org.eclipse.jdt.ls.core/META-INF/MANIFEST.MF

```diff
@@ -31,7 +31,8 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.lsp4j.jsonrpc,
  org.eclipse.xtend.lib,
  org.eclipse.xtext.xbase.lib
-Export-Package: org.eclipse.jdt.ls.core.internal;x-friends:="org.eclipse.jdt.ls.tests",
+Export-Package: org.eclipse.jdt.ls.core,
```

Comment:
The new importer is in a separate bundle and I thought it's better to not reference internals.
Also, until we make a real release (e.g. 1.0) we don't need to commit on the API.